crates/subspace-networking/src/create.rs

```diff
@@ -254,7 +271,8 @@ pub async fn create(config: Config) -> Result<(Node, NodeRunner), CreationError>
 fn build_transport(
     keypair: &identity::Keypair,
     timeout: Duration,
-    yamux_config: YamuxConfig,
+    _yamux_config: YamuxConfig, // TODO: choose yamux or mplex
+    mplex_config: MplexConfig,
```

Comment:
Why are we not supporting both multiplexers at the same time?

Reply:
I'm not follow. They seem mutually exclusive. What will be the benefit of having both working ang glitching muxes in the same configuration?

Reply:
I can introduce some `MuxerChoice` enum for configuration.

Reply:
You can just use either::Either for choosing a muxer in a config

Reply:
They are certainly not mutually exclusive: https://github.com/libp2p/rust-libp2p/blob/62622a1bad70381e56237b4da4d9abcf2b1a0e24/src/lib.rs#L224-L227

Reply:
Hmm, the docs say that the first protocol has a higher priority - this way we could get a  backward compatitbility with `yamux`-only clients.  Ok.